pkgs/applications/terminal-emulators/foot/default.nix

```
@@ -98,7 +98,7 @@ stdenv.mkDerivation rec {
     owner = "dnkl";
     repo = pname;
```

Supersandroing a bit:
```suggestion
     repo = "foot";
```

---

?